### DISCUSSION TITLE
How to draw phosphorylation of a protein by a kinase?

### DISCUSSION THREAD

**Discussion by IsabelWassink** [wikipathways/wikipathways-help#32] [Q&A] [answered]
Hi, I have a question regarding the best way to draw how a kinase phosphorylates a protein. I made a pathway ([WP5358](https://classic.wikipathways.org/index.php/Pathway:WP5358)) based on a [figure](https://pfocr.wikipathways.org/figures/PMC6143874__etm-16-04-3275-g02.html) from the PFOCR project and ran into the following problem: how do I show that a specific protein phosphorylates another protein? So not only that a protein (STAT5A/B) is phosphorylated, but also that the kinase (JAK2) does this. Right now, I used the modification arrow but I am not sure if this is the best option: 
![Schermafbeelding 2023-04-20 om 14 31 47](https://user-images.githubusercontent.com/123644649/233366682-765c7467-e877-48a5-9826-b12e061d2686.png) 
What do you think would be best?

**Comment by khanspers** [comment, marked as answer]
Hi @IsabelWassink, nice work on the pathway! 
Yes, mim-modification is the best (see the MIM specification [here](https://discover.nci.nih.gov/mim/formal_mim_spec.pdf)). However, the arrow in your screenshot looks to be the mim-stimulation arrow, the mim-modification arrow looks like this:
<img width="69" alt="Screen Shot 2023-04-20 at 11 08 16 AM" src="https://user-images.githubusercontent.com/1442197/233451495-af218f3a-ac99-400e-839e-aada9535c28b.png">

That said, as you will see there are many pathways at WikiPathways that simply use the basic default arrow to describe phosphorylation, or even inhibition/stimulation. We have also developed a way of encoding the information about which residue is phosphorylated (if this is known), it's entered as a structured comment on the phospho state. The advantage of entering this information is that it enables data mapping in other applications (i.e. Cytoscape) so that phosphoproteomic data can be mapped to the state. You can see how to create and add the structured comment here: https://academy.wikipathways.org/docs/Modeling%20Phosphorylation%20Site%20Information%20in%20WikiPathways.pdf

**Reply by IsabelWassink** [reply]
Thank you for your answer! I will use the mim-modification arrow from now on.

**Comment by IsabelWassink** [comment]
Hi, I have a follow-up question. I used the mim-modification arrow to show the phosporylation in my pathway ([WP5358](https://classic.wikipathways.org/index.php/Pathway:WP5358)), but there seems to be an error somewhere. In PathVisio, the arrow looks correct and I have checked to make sure it is the modification arrow: 
![Schermafbeelding 2023-04-25 om 13 15 14](https://user-images.githubusercontent.com/123644649/234260309-bd08aa7b-00b7-4bee-8c68-ae2f64521442.png)

However, on WikiPathways (I checked both the classic and new website), the arrow looks like this, even after trying again in PathVisio:
![Schermafbeelding 2023-04-25 om 13 17 28](https://user-images.githubusercontent.com/123644649/234260730-b13e9994-774e-488a-89d3-16a686c4a838.png)
Is there a way I can resolve this issue?

**Comment by khanspers** [comment]
Confirmed. Looks like an svg bug, I have opened an issue [here](https://github.com/wikipathways/wikipathways-development/issues/125).
